.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "api-common-protos"]
```

Comment:
Should this be in here if its in the other CL?

Reply:
It will get handled when I merge from master.